src/engraving/dom/articulation.cpp

```diff
@@ -151,6 +156,28 @@ void Articulation::setUp(bool val)
 //---------------------------------------------------------
 
 TranslatableString Articulation::typeUserName() const
+{
+    if (m_textType != ArticulationTextType::NO_TEXT) {
+        return TranslatableString("engraving", "Articulation text");
+    }
+
+    return TranslatableString("engraving", "Articulation");
+}
+
+String Articulation::translatedTypeUserName() const
+{
+    if (m_textType != ArticulationTextType::NO_TEXT) {
+        return mtrc("engraving", "Articulation Text");
```

Comment:
It would be preferable to use "Sentence case" here too:
```suggestion
        return mtrc("engraving", "Articulation text");
```